rholang/src/test/scala/coop/rchain/rholang/interpreter/SortTest.scala

```diff
@@ -410,39 +410,44 @@ class ParSortMatcherSpec extends FlatSpec with Matchers {
     result.term should be (sortedParExpr)
   }
 
-  "Par" should "Sort Receives based on persistence and then channels and then patterns and then body" in {
+  "Par" should "Sort Receives based on count, persistence, channels, patterns and then body" in {
```

Comment:
You shouldn't need to sort on count. It's utility info.

Reply:
Done.